components/workflow.rst

```diff
@@ -19,6 +19,75 @@ You can install the component in 2 different ways:
 * :doc:`Install it via Composer </components/using_components>` (``symfony/workflow`` on `Packagist`_);
 * Use the official Git repository (https://github.com/symfony/workflow).
 
-For more information, see the code in the Git Repository.
+.. include:: /components/require_autoload.rst.inc
+
+Creating a Workflow
+-------------------
+
+The workflow component gives you an object oriented way to define a process
+or a life cycle that your object goes through. Each step or stage in the
+process is called a *place*. You do also define *transitions* to that describes
+the action to get from one place to another.
+
+.. image:: /_images/components/workflow/states_transitions.png
+
+A set of places and transitions creates a **definition**. A workflow needs
+a ``Definition`` and a way to write the states to the objects, (i.e. an
+instance of a :class:`Symfony\\Component\\Workflow\\MarkingStore\\MarkingStoreInterface`.
+
+Consider the following example for a blog post. A post can have places:
```

Comment:
This -> `A post can have places` looks confusing to me because it uses workflow terminology without a more gentle introduction:

```
A post can have one of a number of predefined statuses (`draft`, `review`, `rejected`, `published`).
In a workflow, these statuses are called **places**.
```